mosec/coordinator.py

```diff
@@ -159,19 +162,31 @@ def run(self):
             self.coordinate()
 
     def get_decoder(self) -> Callable[[bytes], Any]:
-        if STAGE_INGRESS in self.worker._stage:
+        """get the decoder function for this stage
+
+        the first stage will use the worker's deserialize function
+        """
+        if STAGE_INGRESS in self.worker.stage:
             return self.worker.deserialize
-        return self.worker._deserialize_ipc
+        return self.worker.deserialize_ipc
 
     def get_encoder(self) -> Callable[[Any], bytes]:
-        if STAGE_EGRESS in self.worker._stage:
+        """get the encoder function for this stage
```

Comment:
same here and at other places. I think they should have a summary sentence with period at last, and the args list and return list?